daemon/src/peer/run_state.rs

```diff
@@ -138,6 +150,8 @@ pub enum Status {
     Online {
         /// Number of connected peers.
         connected: usize,
```

Comment:
This seems superfluous with the addition of `connected_peers`.

Reply:
Yup, I've removed the whole `connected` field. This will require a few small changes in upstream.